app/views/thredded/posts_common/_before_first_unread_post.html.erb

```diff
@@ -0,0 +1,3 @@
+<% unless post.first_in_page? %>
+  <span id="unread"></span>
```

Comment:
@glebm is there a reason we have to drop this if it's first_in_page? Maybe we could add a class if it's first in page? 

I can imagine wanting to style it differently if it's first in page, but I still would want to be able to have something visible to show that the page is unread. If you think of someone getting used to having a "unread" divider, they might then be expecting it, and searching for it if it's not present, causing confusion.